composer.json

```
@@ -64,6 +64,9 @@
         ],
         "test": [
             "./vendor/bin/pest"
+        ],
+        "pstan": [
+            "./vendor/bin/phpstan"
```

I think it's better to be explicit in CI:s, so maybe add `analyse` here.
Replace with `analyze` depending on English/American (PHPStan supports both). 😅 

```suggestion
            "./vendor/bin/phpstan analyse"
```